src/tests/automaticSettings.test.js

```diff
@@ -855,6 +862,31 @@ describe("options module: AutomaticSettings", function () {
             chai.assert.propertyVal(newOptionFloat, "greatSettingsNum", 12.345);
         });
 
+        it("saves input type=range correctly", async function () {
+            await setupOptionToTest(`
+            <li><label for="exampleId">exampleId</label>
+            <input class="setting save-on-change" type="range" value="0" min="0" max="10" step="1" id="exampleId" name="exampleOption">
+            </li>`, "exampleOption", "exampleId", 4);
```

Comment:
```suggestion
            </li>`, "exampleOption", "exampleId", 8);
```

Not tested, but I guess this should fix it.